idea_town/accounts/providers/fxa/views.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+from allauth.socialaccount.providers.oauth2.views import (OAuth2Adapter,
+                                                          OAuth2LoginView,
+                                                          OAuth2CallbackView)
+from .provider import FirefoxAccountsProvider
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class FirefoxAccountsOAuth2Adapter(OAuth2Adapter):
+    provider_id = FirefoxAccountsProvider.id
+
+    def __init__(self):
+        provider = self.get_provider()
+        provider_settings = provider.get_settings()
+
+        self.access_token_url = provider_settings.get(
+            'ACCESS_TOKEN_URL', 'https://oauth.accounts.firefox.com/v1/token')
```

Comment:
yeah, it looks like we are duplicating since we are passing in the same defaults as are already in SOCIALACCOUNT_PROVIDERS